### PR TITLE
feat: remove compass, upgrade to node-sass

### PR DESCRIPTION
**Pull request by asimpson**
:eyes: @robtarr?

### CODE REVIEW

**Comment by asimpson** [comment]
or @patricksimpson :smile:

File: app/templates/_package.json [hunk 1]
```diff
@@ -19,7 +19,7 @@
     "grunt-contrib-connect": "~0.5.0",
     "connect-livereload": "~0.3.0",
     "grunt-grunticon": "~1.0.0",
-    "grunt-bower": "~0.7.0",
+    "grunt-bower": "^0.18.0",
```
Comment (cromwellryan) [inline app/templates/_package.json]: Do we want to be so lenient on version matching?  `^` matches will any 0.* version vs `~` matching any 0.18.* version.

Comment (cromwellryan) [inline app/templates/_package.json]: Do we want to be so lenient with version matching?  `^` matches any 0.* version while `~` matches 0.18.*.

**Comment by cromwellryan** [comment]
Other than the version match it worked like charm.

**Comment by asimpson** [comment]
Good catch, I overreacted to it being out of date and breaking everything. I'll change that once I get in this morning. Thanks man :+1:

**Comment by robtarr** [comment]
The readme is still referencing the SASS gem

**Comment by asimpson** [comment]
Good call :bow:

**Comment by asimpson** [comment]
@robtarr @cromwellryan good?

**Pull request closed without merging**